documentation/example-module-py/example-module.py

```diff
@@ -1,20 +1,19 @@
 import yarp
 import sys
 import numpy as np
-import matplotlib.pyplot as plt
 import threading
+import cv2
 
 
-class ExampleModule(yarp.RFModule, threading.Thread):
+class ExampleModule(yarp.RFModule):
```

Comment:
this class doesn't seem to inherit from yarp.thread but as the asynchronous thread `run()` - is it correct?

Reply:
@Iaxama - do you get a notification if I make a comment?

Reply:
yes that's correct because we spawn the thread in the configure, without the need of extending the thread class